qemu/tests/drive_mirror.py

```diff
@@ -66,6 +66,22 @@ def get_device(self):
         return self.vm.get_block({"file": image_file})
 
     @error.context_aware
+    def check_granularity(self):
+        """
+        Check granularity value as set.
+        """
+        target_size = int(self.params.get("granularity"))
```

Comment:
please use params["granularity"], so that ParamError  exception will raise when leak this param